.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Read this First**
+If you have a Trusted Web Activity related question, the best place to ask it is on StackOverflow, on the
+[trusted-web-activity](https://stackoverflow.com/questions/tagged/trusted-web-activity) tag, which is also monitored by the team.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. Pixel 3]
+ - OS: [e.g. Android 10]
+ - Browsers Installed [e.g. Chrome, Firefox, etc]
+ - Browser Versions [e.g. 22]
+ - android-browser-helper library version [eg: 1.1.0]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
We could add a "Did this ever used to work" to determine if something is a regression.

Also, maybe prompt for code snippets/links to a public repo.

Reply:
Added "Did this ever work" and a section where developers can drop code snippets or link.